docs/en/integrations/language-clients/js.md

```diff
@@ -754,14 +830,15 @@ class MyLogger implements Logger {
 createClient({
   log: {
     LoggerClass: MyLogger,
+    level: ClickHouseLogLevel
   }
 })
 ```
 
 Check an example implementation
 [here](https://github.com/ClickHouse/clickhouse-js/blob/3aad886231e93c982b0c6e552c87ce7fa72c2caf/__tests__/utils/test_logger.ts#L4-L17).
 
-## TLS certificates
+## TLS certificates (Node.js only)
```

Comment:
IMO mixing documentaiton is a bit confusing. Maybe we should organize it in a few sections:
- common API
- nodejs API
- web API

@DanRoscigno do you have any suggestions?